blackbox/utils/reports.py

```diff
@@ -3,12 +3,16 @@
 
 @dataclasses.dataclass
 class StorageReport:
+    """Keep storage output."""
```

Comment:
Better here to describe what this is used to represent.

```suggestion
    """A report created by one of the Storage Handlers."""
```